src/poliastro/constants.py

```diff
@@ -335,3 +335,12 @@
     "Howard Curtis",
     system="si",
 )
+
+altitude_GEO = Constant(
+    "altitude_GEO",
+    "altitude of geosynchronous equatorial(geostationary) orbit",
+    3.5786e7,
+    "m",
+    1,
+    "Wikipedia",
```

Comment:
Sure, I will do that. Thanks for the review :)
`black`  Is that a package I need to install?

Reply:
Yes! you may `pip install black` . And you may also consider a rebase to current master!

Reply:
Also, I am not sure if we can use data from `Wikipedia` . (I have never seen any other reference to it) For that @Juanlu001 can tell. 
After this, you will also have to add a test for the method.

Reply:
You can copy paste the code in notebook, and then call the function!

Reply:
@hrishikeshgoyal take a look at https://github.com/poliastro/poliastro/blob/master/CONTRIBUTING.rst#development-environment in case you didn't before, and use `pytest -k geostationary` to run only tests with "geostationary" in the name.